components/common/icon-rectangle-box/RectangleBoxIconInverse.tsx

```diff
@@ -0,0 +1,13 @@
+const RectangleBoxIconInverse = () => {
+  return (
+    <svg width="80" height="80" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Could you use the `SvgIcon` component from MUI?

Reply:
'+